src/app/header/header.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div class="navbar-header">
+    <a href="#" class="navbar-brand">
+      <div class="user-records">
+        <div class="icon-text">
+          <mat-icon class="icon" svgIcon="user"></mat-icon>
```

Comment:
Instead of using this `mat-icon` component, can't you directly use the svg icon?

Reply:
Yes it can be done, and I have now used this method since it is much simpler.